package.json

```diff
@@ -23,7 +23,7 @@
     "bootstrap-sass": "^3.3.7",
     "cross-env": "^5.2.0",
     "husky": "^2.7.0",
-    "jquery": "^3.4.1",
+    "jquery": "^3.5.0",
```

Comment:
@DFurnes since a minor bump this should be alright. Thoughts?

Reply:
Yep! Seems like a pretty [minor change](https://blog.jquery.com/2020/04/10/jquery-3-5-0-released/).